app/views/subnets/_fields.html.erb

```diff
@@ -22,6 +22,7 @@
   <%= text_f f, :to,   :help_inline => _("Optional: Ending IP Address for IP auto suggestion") %>
 </div>
 <%= number_f f, :vlanid, :min => 0, :max => 4095, :help_inline => _("Optional: VLAN ID for this subnet") %>
+<%= number_f f, :mtu, :min => 68, :max => 4294967295, :help_inline => _("Optional: MTU for this subnet") %>
```

Comment:
Is it really optional? I think this is kind of required as the database has a default value.

Reply:
Done.